autogen/extensions/streaming_transmitter.py

```diff
@@ -0,0 +1,9 @@
+from typing import Protocol
+
+
+class StreamingTransmitter(Protocol):
+    def __init__(self, transmitter):
```

Comment:
Why do you want to specify the constructor? There is no need for this, classes implementing the protocols can be constructed any way they see fit, you just need the `send` method.